Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveM.cpp

```diff
@@ -43,6 +43,7 @@ void FiniteDifferenceSolver::MacroscopicEvolveM (
         amrex::Real const dt,
         std::unique_ptr<MacroscopicProperties> const& macroscopic_properties )
     {
+    amrex::Real mag_normalized_error = 0.1; // normalization error of M field for checking
```

Comment:
Can we make ``mag_normalized_error`` a user-input?

Reply:
I'm fine if we make this a very short, separate PR as well.